includes/transformer/class-transformers-manager.php

```diff
@@ -0,0 +1,292 @@
+<?php
+/**
+ * Inspired by the way elementor handles addons.
+ *
+ * @link https://github.com/elementor/elementor/
+ * @package Activitypub
+ */
+
+namespace Activitypub\Transformer;
+
+use WP_Post;
+use WP_Comment;
+
+use function Activitypub\camel_to_snake_case;
+use function Activitypub\snake_to_camel_case;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * ActivityPub transformers manager.
+ *
+ * ActivityPub transformers manager handler class is responsible for registering and
+ * initializing all the supported WP-Pobject to ActivityPub transformers.
+ *
+ * @since version_number_transformer_management_placeholder
+ */
+class Transformers_Manager {
+	const DEFAULT_TRANSFORMER_MAPPING = array(
```

Comment:
there are a lot of function/attributes that are already defined by the Base object.